frontend/public/locales/fr/translation.json

```
@@ -116,6 +116,10 @@
   },
   "tabsBox": {
     "errorOnLoading": "Une erreur est survenue lors du chargement des résultats.",
+    "rateLater": "Vos vidéos à comparer plus tard apparaissent ici. Vous pouvez en ajouter sur le site web via le bouton '+'. Vous pouvez aussi les ajouter directement depuis YouTube grâce à <2>l'extension</2>.",
+    "compared": "Les vidéos comparées apparaissent ici.",
+    "recommendations": "Les vidéos recommandées apparaissent ici.",
```

Similarly to the english version, these messages could be made more explicit:

```suggestion
    "compared": "Les vidéos que vous avez comparées apparaissent ici.",
    "recommendations": "Les vidéos recommandées par la communauté apparaissent ici.",
```